media-sound/jack2/jack2-1.9.11_rc1.ebuild

```diff
@@ -0,0 +1,82 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="5"
+
+PYTHON_COMPAT=( python2_7 )
+PYTHON_REQ_USE="threads(+)"
+inherit eutils python-single-r1 waf-utils multilib-minimal
+
+DESCRIPTION="Jackdmp jack implemention for multi-processor machine"
+HOMEPAGE="http://jackaudio.org/"
+
+RESTRICT="mirror"
+if [[ "${PV}" = "2.9999" ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="git://github.com/jackaudio/jack2.git"
+	KEYWORDS=""
+else
+	MY_PV="${PV/_rc/-RC}"
+	MY_P="${PN}-${MY_PV}"
+	S="${WORKDIR}/${MY_P}"
+	SRC_URI="https://github.com/jackaudio/jack2/archive/v${MY_PV}.tar.gz -> ${MY_P}.tar.gz"
+	KEYWORDS="~amd64 ~ppc ~x86"
+fi
+
+LICENSE="GPL-2"
+SLOT="2"
+IUSE="alsa celt dbus doc opus pam"
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+# FIXME: automagic deps: readline, samplerate, sndfile, celt, opus
+# FIXME: even though sndfile is just used for binaries, the check is flawed
+#	making the build fail if multilib libsndfile is not found.
+CDEPEND="media-libs/libsamplerate[${MULTILIB_USEDEP}]
+	media-libs/libsndfile[${MULTILIB_USEDEP}]
+	sys-libs/readline:0
+	${PYTHON_DEPS}
+	alsa? ( media-libs/alsa-lib[${MULTILIB_USEDEP}] )
+	celt? ( media-libs/celt:0[${MULTILIB_USEDEP}] )
+	dbus? (
+		dev-libs/expat[${MULTILIB_USEDEP}]
+		sys-apps/dbus[${MULTILIB_USEDEP}]
+	)
+	opus? ( media-libs/opus[custom-modes,${MULTILIB_USEDEP}] )"
+DEPEND="!media-sound/jack-audio-connection-kit:0
+	${CDEPEND}
+	virtual/pkgconfig
+	doc? ( app-doc/doxygen )"
+RDEPEND="${CDEPEND}
+	dbus? ( dev-python/dbus-python[${PYTHON_USEDEP}] )
+	pam? ( sys-auth/realtime-base )"
+
+DOCS=( ChangeLog README README_NETJACK2 TODO )
+
+src_prepare() {
+	default
+	multilib_copy_sources
+}
+
+multilib_src_configure() {
+	local mywafconfargs=(
+		--htmldir=/usr/share/doc/${PF}/html
+		$(usex alsa --alsa=yes --alsa=no)
+		$(usex dbus --dbus --classic)
+		$(usex doc --doxygen=yes --doxygen=no)
+	)
```

Comment:
@aballier do you want to add all of those? I added them locally but removed them again because I didn't want to introduce all these changes just for revbump, the majority of them alread existed in the previous version https://github.com/jackaudio/jack2/blob/v1.9.10/wscript